src/pages/SignUp.vue

```diff
@@ -1,14 +1,99 @@
 <template>
   <q-page padding>
-    <h1>Sign Up Page</h1>
+    <h1>Registration</h1>
+    <form>
+        <q-input style="padding: 10px" v-model="email" outlined label="Email"
+          @blur="emailCheck" :error="$v.email.$error"/>
+        <q-input style="padding: 10px" v-model="username" outlined label="UserName"
+          @blur="usernameCheck" :error="usernameError"/>
+        <q-input style="padding: 10px" v-model="pwd" outlined label="Password"
+          @blur="passwordCheck" type="password" :error="pwdError"/>
+        <q-input style="padding: 10px" v-model="cPwd" outlined label="Confirm Password"
+          @blur="passwordMatchCheck" :error="cPwdError" type="password"/>
+    </form>
+    <div class="row justify-center" style="margin: 30px">
+      <q-btn @click="submit" color="primary" label="Create Account"  />
```

Comment:
```suggestion
      <q-btn type="submit" color="primary" label="Create Account"  />
```
Also, make sure to move this button (and probably its parent `<div>`) inside the `<form>`.